projects/Mallard/src/components/article/index.tsx

```diff
@@ -67,7 +68,7 @@ const Article = ({
     standfirst,
 }: {
     navigation: NavigationScreenProp<{}>
-    article: any[]
+    article: IBlockElement[]
```

Comment:
We should be using [`BlockElement`](https://github.com/guardian/editions/blob/master/projects/backend/common.ts#L66) here, but we'll also need to update Laura's Funhouse 🤡 🏠 🤡